app/src/main/java/com/example/theperiodpurse/ui/calendar/CalendarScreen.kt

```diff
@@ -51,56 +80,190 @@ fun Tabs(tabs: List<CalendarTabItem>, pagerState: PagerState) {
         CalendarTabItem.CycleTab
     )
     val pagerState = rememberPagerState()
-    ThePeriodPurseTheme() {
+    ThePeriodPurseTheme {
         Scaffold (topBar = {})
         { padding ->
             Column(modifier = Modifier.padding(padding)) {
                 Tabs(tabs = tabs, pagerState = pagerState)
-                TabsContent(tabs = tabs, pagerState = pagerState)
+                TabsContent(tabs = tabs, pagerState = pagerState, navController = navController)
             }
         }
     }
 }
 
+@RequiresApi(Build.VERSION_CODES.O)
 @Composable
-fun CalendarScreenLayout() {
-    // Contains the swippable content
-    ThePeriodPurseTheme() {
+fun CalendarScreenLayout(navController: NavController) {
+    // Contains the swappable content
+    ThePeriodPurseTheme {
+        val bg = painterResource(R.drawable.colourwatercolour)
+
+        var selectedDate by remember { mutableStateOf<LocalDate?>(null) }
+        val currentMonth = remember { YearMonth.now() }
+        val startMonth = remember { currentMonth.minusMonths(12) } // Previous months
+        val endMonth = remember { currentMonth.plusMonths(0) } // Next months
+        val firstDayOfWeek = remember { firstDayOfWeekFromLocale() } // Available from the library
+
+        val state = rememberCalendarState(
+            startMonth = startMonth,
+            endMonth = endMonth,
+            firstVisibleMonth = currentMonth,
+            firstDayOfWeek = firstDayOfWeek
+        )
+
+        Box {
+            Image(painter = bg,
+                contentDescription = null,
+                modifier = Modifier
+                    .fillMaxSize()
+                    .semantics { contentDescription = "Calendar Page" },
+                contentScale = ContentScale.FillBounds,
+            )
+            Column {
+                VerticalCalendar(
+                    modifier = Modifier.semantics { contentDescription = "Calendar" },
+                    state = state,
+                    monthHeader = { month ->
+                        MonthHeader(month) },
+                    dayContent = { day ->
+                        Day(day, isSelected = selectedDate == day.date) { day ->
+                            selectedDate = if (selectedDate == day.date) null
+                            else day.date
+                            navController.navigate(
+                                    route = "%s/%s/%s"
+                                        .format(
+                                            Screen.Calendar,
+                                            Screen.Log,
+                                            day.date.toString()
+                                        )
+                                )
+                        }
+                    }
+                )
+            }
+        }
+    }
+}
+
+// Creates the days
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun Day(day: CalendarDay,
+        isSelected: Boolean,
```

Comment:
remove unused

Reply:
Unfortunately, I can't remove the isSelected parameter since it's required for the `dayContent ` parameter for the Calendar so I'd have to keep the parameter there even though it gives that warning.